pkg/testing/functional.go

```diff
@@ -22,98 +22,91 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	"knative.dev/serving/pkg/apis/autoscaling"
-	autoscalingv1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
+	asv1a1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
```

Comment:
Most of the project uses `av1alpha1` as the shorthand for the package, maybe that's the convention we should follow?

Reply:
This feels shorter. I'd rather we update other places.

Reply:
> I'd rather we update other places.

Then this PR is incomplete isn't it? ;-)

Reply:
This is one way to look at it.
Each time I touch a file, I fix it :). So it's a process.